src/visualize/image_like.jl

```diff
@@ -127,35 +109,114 @@ function play{T}(buffer::Array{T, 2}, video_stream, t)
     return reinterpret(T, buffer, (w,h))
 end
 
-"""
-Takes a 3D image and decides if it is a volume or an animated Image.
-"""
-function _default{T<:Colorant, X}(img::Images.Image{T, 3, X}, s::Style, data::Dict)
-    props = img.properties
-    if haskey(props, "timedim")
-        if haskey(props, "spatialorder")
-            get!(data, :spatialorder, join(props["spatialorder"], ""))
+# If the user is using the new Images, ImageAxes will be loaded
+if isdefined(Images, :ImageAxes)
+    unwrap(img::Images.ImageMeta) = unwrap(data(img))
```

Comment:
Okay let me see if I can materialize this in code... One problem is on 0.5, that `Signal{AbstractArray{T, N}}` doesn't dispatch... Let's see what I can do about it.